core/src/main/java/org/apache/calcite/sql/SqlKind.java

```diff
@@ -988,17 +988,21 @@
   /** The {@code ROW_NUMBER} window function. */
   CUME_DIST,
 
-  // Group functions
+  /** The {@code DESCRIPTOR}. */
+  DESCRIPTOR,
 
-  /** The {@code TUMBLE} group function. */
+  /** The {@code TUMBLE_TABLE_FUNCTION}. */
```

Comment:
I thought Sqlkind.TUMBLE will be dedicated to the built-in table-valued function. Given another thought now, since the old TUMBLE is not deprecated yet, so I roll back this change to make SqlKind.TUMBLE still be used to refer the old TUMBLE.

We can change all the comments together when be ready to deprecate old TUMBLE.

Reply:
My point was that TUMBLE_TABLE_FUNCTION is not a SQL keyword, so it shouldn't be "code".

Ideally TUMBLE becomes an ordinary table function. If so, we can stop using SqlKind.TUMBLE as a special case. If not, that's also fine.